orm/src/test/java/org/hibernate/search/test/bridge/BridgeTest.java

```diff
@@ -359,6 +360,74 @@ public void testCalendarBridge() throws Exception {
 	}
 
 	@Test
+	@Category(SkipOnElasticsearch.class) // Elasticsearch uses a specific encoding for dates
+	@SkipForDialect(PostgreSQL81Dialect.class)//PosgreSQL doesn't allow storing null with these column types
+	public void testDateBridgeStringEncoding() throws Exception {
+		Calendar c = Calendar.getInstance( TimeZone.getTimeZone( "Europe/Rome" ), Locale.ROOT ); //for the sake of tests
+		c.set( 2000, Calendar.DECEMBER, 15, 3, 43, 2 );
+		c.set( Calendar.MILLISECOND, 5 );
+		Date date = new Date( c.getTimeInMillis() );
+
+		Cloud cloud = new Cloud();
+		cloud.setDateDay( date );
+
+		org.hibernate.Session s = openSession();
+		Transaction tx = s.beginTransaction();
+		s.persist( cloud );
+		s.flush();
+		tx.commit();
+
+		tx = s.beginTransaction();
+		FullTextSession session = Search.getFullTextSession( s );
+
+		BooleanQuery.Builder booleanQueryBuilder = new BooleanQuery.Builder();
+
+		TermQuery termQuery = new TermQuery( new Term( "dateDayStringEncoding",
+				DateTools.dateToString( date, DateTools.Resolution.DAY ) ) );
+		booleanQueryBuilder.add( termQuery, BooleanClause.Occur.MUST );
```

Comment:
you shouldn't need to wrap it in a boolean query "MUST when it's the only clause?

I'll simplify both methods.

Reply:
Right, I'll rebase and fix that

Reply:
Ah, didn't see your comment. Thanks :)